Packages.Data.props

```diff
@@ -33,7 +33,7 @@
   <!-- All csproj dependencies. -->
   <ItemGroup>
     <PackageReference Update="NUnit" Version="3.13.2" />
-    <PackageReference Update="System.ClientModel" Version="1.0.0-beta.3" />
+    <PackageReference Update="System.ClientModel" Version="1.2.0-alpha.20240311.5" />
```

Comment:
Why are we using an alpha here?

Reply:
I spoke with @KrzysztofCwalina on this and as a temporary transition to the new 1.1.0 GA this is fine.